src/com/dd/levels/Room.java

```diff
@@ -19,32 +19,24 @@ public String examineString() {
 		StringBuilder examineStr = new StringBuilder();
 		if(isEmpty()){
 			examineStr.append("This room is empty.");
+			return examineStr.toString();
 		}
-		if(hasMonster()) {
-			if(getMonsterList().size() == 1) {
-				examineStr.append("This room has a ");
-				getMonsterList().forEach((k) -> examineStr.append(k + ".\n"));
-			}
-			else {
-				examineStr.append("This room has multiple monsters:");
-				getMonsterList().forEach((k) -> examineStr.append("\n~" + k + "\n"));
-			}
-			examineStr.append("Time to fight!\n");
+		if(hasMonster()) {	
+			examineStr.append("This room has a ");
+			getMonsterList().forEach((k) -> examineStr.append(monsterMap.get(k).typeToString() + " named \"" + k + "\" "));
```

Comment:
There will be two descriptions, one for when you enter a room with a monster and one for when you use examine monster/room